bokehjs/test/document.coffee

```diff
@@ -553,6 +553,10 @@ describe "Document", ->
     out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
     expect(out).to.be.equal ""
 
+    parsed['version'] = "#{js_version}rc123"
+    out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
+    expect(out).to.be.equal ""
```

Comment:
Breaking it down another way, if the python version is of the form:
* `0.1.2`
* `0.1.2dev3`
* `0.1.2rc3` 
Then both python and JS versions should match *exactly* and warn if they don't. The only time to skip is if the python version is somethign like `0.1.2-3-ae56af` (i.e., has a dash in it). In that case the test should just be skipped. Looking at the test above, I'd expect it to fail.